src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -237,6 +239,13 @@ export class CalciteRadioButton implements LabelableComponent, CheckableFormComp
     });
   }
 
+  private getTabIndex(): number {
+    if (this.disabled) {
+      return;
```

Comment:
I think this should return -1. Can probably be included in the return statement below too.

Reply:
See my earlier question about disabled inputs.

Reply:
Based on the updated thread on the earlier question, can you explicitly return `undefined` and update the return type?